src/components/mainMenu/mainMenu.js

```diff
@@ -105,7 +105,7 @@ class MainMenu extends React.Component {
     ];
 
     if (showDelegate) {
-      tabs.splice(tabs.length - 2, 0, {
+      tabs.splice(tabs.length - 1, 0, {
```

Comment:
It's not the first time we have a problem with the index here, I would rather put delegates item inside `tabs` array and `.filter` it by `id === 'delegates'` instead of `splice`ing based on the index.

Reply:
👍